src/sql/src/plan/query.rs

```diff
@@ -147,39 +147,44 @@ pub fn plan_insert_query(
         bail!("cannot insert into system table '{}'", table.name());
     }
 
+    // Normalize column names
```

Comment:
You can probably skip this comment - I think the next line is clear!

Reply:
You're right, I'll remove it :)

Reply:
fixed